templates/manage.html

```
@@ -5,147 +5,211 @@
+                    <div class="workspace-head">
+                        <div>
+                            <div class="eyebrow">New host</div>
+                            <h2>Add server</h2>
+                        </div>
+                        <p class="muted workspace-note">Auth is password or SSH key; a per-server key takes priority.</p>
                     </div>
-                </div>
-            </form>
+                    <form id="add-server-form" class="add-server-form">
+                        <span class="muted form-note">Leave password and key empty only when the global SSH key should be used.</span>
+                        <input type="text" id="name" placeholder="Name" required>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)